QueueInteropTransport.php

```diff
@@ -137,6 +139,10 @@ public function send(Envelope $envelope): Envelope
 
         $producer = $context->createProducer();
 
+        if (null !== $envelope->last(RedeliveryStamp::class) && $producer instanceof SnsQsProducer) {
+            $topic = $context->createQueue($destination['queue']);
```

Comment:
Sorry for my ignorance. Is `$destination['queue']` always defined?

Reply:
It should be, but better add a check anyway, thanks!